src/main/java/uk/gov/companieshouse/web/pps/controller/BaseController.java

```diff
@@ -18,14 +18,33 @@ public abstract class BaseController {
 
     protected static final String ERROR_VIEW = "error";
 
+    public static final String BACK_LINK_ATTR = "backLink";
+    public static final String USER_BAR_ATTR = "userBar";
+    public static final String USER_SIGNOUT_URL_ATTR = "userSignoutUrl";
+    public static final String HIDE_YOUR_DETAILS_ATTR = "hideYourDetails";
+    public static final String HIDE_RECENT_FILINGS_ATTR = "hideRecentFilings";
+    public static final String PHASE_BANNER_ATTR = "phaseBanner";
+    public static final String PHASE_BANNER_LINK_ATTR =  "phaseBannerLink";
+
     protected BaseController() {
     }
 
     @ModelAttribute("templateName")
     protected abstract String getTemplateName();
 
     protected void addBackPageAttributeToModel(Model model, String... pathVars) {
```

Comment:
Is it worth having a single method call that adds the back page, user model and phase banner all together, so when we call it in the other controllers, we can use one method call instead of 3?]

For example;
```suggestion
    protected void  addBaseAttributesToModel() {
       addBackPageAttributeToModel();
       addUserModel();
       addPhaseBannerToModel()
```

That way we can access these in single place and then make methods private.

Although this won't work if we have pages where only one of these attributes is being added, we could maybe use the single method calls in those places.

Reply:
Updated that with the suggestion. created one for all 3 and other one without the UserBar.